### ISSUE TITLE
fman fails to start on Fedora 29

### ISSUE THREAD

**Issue by mherrmann** [fman-community/fman#561]
It works on Fedora 28. On 29, you get

 > This application failed to start because it could not find or load the Qt platform plugin "xcb" in "".
 > Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.
 > Reinstalling the application may fix this problem.

A temporary workaround is to start fman with the following command:

    LD_PRELOAD=/lib64/libfreetype.so /opt/fman/fman

**Comment by brunojm** [comment]
Also, when we run the 'Install plugin' command we get this:

```Command 'InstallPlugin' raised error.

Traceback (most recent call last):
  File "fman/impl/plugins/command_registry.py", line 66, in _execute_command
  File "/opt/fman/Plugins/Core/core/commands/__init__.py", line 1572, in __call__
    find_repos(topics=['fman', 'plugin'])
  File "/opt/fman/Plugins/Core/core/github.py", line 13, in find_repos
    return list(map(GitHubRepo, _fetch_all_pages(url)))
  File "/opt/fman/Plugins/Core/core/github.py", line 17, in _fetch_all_pages
    data = _get_json(json_url + '&per_page=%d&page=%d' % (page_size, page))
  File "/opt/fman/Plugins/Core/core/github.py", line 72, in _get_json
    return json.loads(_get(url).decode('utf-8'))
  File "/opt/fman/Plugins/Core/core/github.py", line 76, in _get
    return urlopen(url).read()
  File "urllib/request.py", line 163, in urlopen
  File "urllib/request.py", line 466, in open
  File "urllib/request.py", line 484, in _open
  File "urllib/request.py", line 444, in _call_chain
  File "urllib/request.py", line 1297, in https_open
  File "urllib/request.py", line 1223, in do_open
  File "http/client.py", line 1238, in __init__
  File "ssl.py", line 449, in create_default_context
  File "ssl.py", line 369, in __new__
ssl.SSLError: ('failed to allocate SSL context',)```

**Comment by mherrmann** [comment]
I just released [fman 1.4.3](https://fman.io/changelog#v1.4.3) that now does start on Fedora. I'll open a new issue for the failing `InstallPlugin` command.
